libbeat/tests/system/beat/beat.py

```diff
@@ -683,7 +683,9 @@ def is_documented(key, docs):
 
         for key in flat.keys():
             metaKey = key.startswith('@metadata.')
-            if not(is_documented(key, expected_fields) or metaKey):
+            # Range keys as used in 'date_range' etc will not have docs of course
+            isRangeKey = key.split('.')[-1] in ['gte', 'gt', 'lte', 'lt'];
```

Comment:
@ruflin are you OK with this change before I merge?

Reply:
yes.